bin/run_tests

```diff
@@ -14,4 +14,5 @@ docker-compose -f docker/compose/docker-compose.yaml run --rm transact \
    /bin/bash -c \
    "cargo test && \
     cargo test --manifest-path /project/transact/libtransact/Cargo.toml --features sawtooth-compat && \
-    cargo test --manifest-path /project/transact/libtransact/Cargo.toml --features experimental"
+    cargo test --manifest-path /project/transact/libtransact/Cargo.toml --features experimental && \
+    cargo test --manifest-path /project/transact/libtransact/Cargo.toml --no-default-features --features state-merkle-redis-db-tests"
```

Comment:
This should probably be run with experimental features so we know its compatible with other experimental changes (and not run it at all with --no-default-features).

Reply:
I did catch a dependency issue by doing this, but I can add back experimental features.